testsuite/features/step_definitions/command_steps.rb

```diff
@@ -696,6 +696,25 @@
   node.run(cmd, error_control.empty?)
 end
 
+When(/^I (enable|disable) the repositories "([^"]*)" on this "([^"]*)"((?: without error control)?)$/) do |action, repos, host, error_control|
```

Comment:
Could by any chance this new step be reused elsewhere?

Reply:
Yes, there are several places where we enable multiple repos one by one.
I just didn't change them inside the same PR, so the risk is lower.
But fine to so it in another PR tomorrow.

The main purpose of this PR was to fix the monitoring test feature in Uyuni